build-jenkins-agent-ubuntu.pkr.hcl

```diff
@@ -42,7 +37,12 @@ build {
 
   provisioner "shell" {
     environment_vars = local.provisioning_env_vars
-    execute_command  = "chmod +x {{ .Path }}; {{ .Vars }} sudo -E bash '{{ .Path }}'"
     script           = "./provisioning/ubuntu-provision.sh"
```

Comment:
```suggestion
    execute_command  = "chmod +x {{ .Path }}; {{ .Vars }} sudo -E bash '{{ .Path }}'"
    script           = "./provisioning/ubuntu-provision.sh"
```

Restoring this to avoid error when building amazon-ebs images.

<img width="1456" alt="image" src="https://user-images.githubusercontent.com/91831478/180817938-28c6a330-733c-4fa1-abf6-369bd653f565.png">